pkg/network/kubemacpool.go

```diff
@@ -40,6 +41,19 @@ func validateKubeMacPool(conf *opv1alpha1.NetworkAddonsConfigSpec) []error {
 		return []error{errors.Errorf("failed to parse rangeEnd because the mac address is invalid")}
 	}
 
+	if err := checkRange(net.HardwareAddr(conf.KubeMacPool.RangeStart), net.HardwareAddr(conf.KubeMacPool.RangeEnd)); err != nil {
```

Comment:
Could you mention in the commit message that you added this validation, not only the test for it?